application/wso2verify/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Add the license, do the same for all the files

Reply:
Done. Check the latest commit